pynumaflow/sinker/servicer/utils.py

```diff
@@ -0,0 +1,13 @@
+from pynumaflow.proto.sinker import sink_pb2
+from pynumaflow.sinker._dtypes import Response
+
+
+def build_sink_response(rspn: Response):
+    if rspn.fallback:
+        return sink_pb2.SinkResponse.Result(id=rspn.id, status=sink_pb2.Status.FALLBACK)
+    elif rspn.success:
+        return sink_pb2.SinkResponse.Result(id=rspn.id, status=sink_pb2.Status.SUCCESS)
```

Comment:
NIT: order based on the likely path, rspn.success as first, rest doesn't matter

Reply:
You got it boss!